style.css

```diff
@@ -62,6 +62,102 @@ body {
   border: 1px solid #6f6c6b;
 }
 
+.popup-container {
+  position: fixed;
+  width: 100%;
+  height: 100vh;
+  background-color: rgba(151, 148, 147, 0.9);
+  box-sizing: border-box;
+  overflow-y: scroll;
+  top: 0;
+  left: 0;
+}
+
+.popup-details-container {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  background-color: #fff;
+  padding: 16px;
+  margin: 60px 16px 39px 16px;
+}
+
+.popup-name-image {
+  display: flex;
+  justify-content: space-between;
+}
+
+.popup-name {
+  color: #172b4d;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.popup-image {
+  height: 40px;
+  width: 40px;
+}
```

Comment:
- [ ] [OPTIONAL] You have done great work 👍 , your pop-up looks amazing ✨  and everything works 👌  however I would suggest you use relative values foe sizes like %, em, rem instead of absolute size like px. Unlike PX, relative units like %, EM, and REM are better suited to responsive design and also help meet accessibility standards. Relative units scale better on different devices because they can scale up and down according to another element's size.